src/main/resources/assets/forge/lang/en_US.lang

```diff
@@ -182,4 +182,6 @@ fml.messages.version.restriction.bounded=between %s and %s
 fml.button.visitjavadownloads=Oracle Java SE Downloads
 fml.button.continue=Continue
 fml.button.open.mods.folder=Open Mods Folder
-fml.button.open.file=Open %s
\ No newline at end of file
+fml.button.open.file=Open %s
+
+forge.container.enchant.enchantabilityTooLow=Enchantability Low
```

Comment:
Something along the lines of "Item has limited enchantments"? I feel like you did a good job coming up with something short but it doesn't fully convey what's going on.